src/types/metaAggregator.ts

```diff
@@ -0,0 +1,50 @@
+import { Currency, CurrencyAmount, Price } from '@kyberswap/ks-sdk-core'
+
+export type Route = {
+  pool: string
+
+  tokenIn: string
+  swapAmount: string
+
+  tokenOut: string
+  amountOut: string
+
+  limitReturnAmount: string
+  exchange: string
+  poolLength: number
+  poolType: string
+  extra: string
+}
+
+export type FeeConfig = {
+  feeAmount: string
+  chargeFeeBy: string
+  isInBps: boolean
+  feeReceiver: string
+}
+
+export type RouteSummary = {
```

Comment:
Nah, it's intentional. I'd like to separate from the type of response and type of the data that we use. This data is parsed from the response and it can have a few more fields.
There's duplication but if the response changes, we need to update the type of response and the parse function only.